src/main/kotlin/de/cyface/uploader/Uploader.kt

```diff
@@ -68,11 +67,10 @@ interface Uploader {
     @Throws(UploadFailed::class)
     fun uploadAttachment(
         jwtToken: String,
-        metaData: RequestMetaData,
-        measurementId: Long,
+        uploadable: Attachment,
         file: File,
         fileName: String,
-        progressListener: UploadProgressListener
+        progressListener: UploadProgressListener,
```

Comment:
I started putting a comma after the last parameter in such multi-line formats, so when we add or remove a parameter only the added line is shown in git as changed, not the previously last line where we would just have to add a `,` at the end.

just fyi